fuzzers/pip_loop.mk

```diff
@@ -30,10 +30,10 @@ all: database
 
 $(SPECIMENS_OK): build/todo.txt $(SPECIMENS_DEPS)
 	mkdir -p build/$(ITER)
-	if [ -f ${FUZDIR}/generate.sh ] ; then \
-			bash ${FUZDIR}/generate.sh $(subst /OK,,$@) ; \
+	+if [ -f ${FUZDIR}/generate.sh ] ; then \
+			+bash ${FUZDIR}/generate.sh $(subst /OK,,$@) ; \
```

Comment:
I don't think you should be using `+` at the second level here?

Reply:
I didn't see the BRAM fuzzer locally, but will see how this behaves on CI without the second ``+``